exporters/committime/__init__.py

```diff
@@ -124,12 +126,53 @@ def git_server(self):
 
         return url
 
+    @property
+    def azure_project(self):
+        return self.__azure_project
+
     def __parse_repourl(self):
         """Parse the repo_url into individual pieces"""
         logging.debug("repo url = %s", self.__repo_url)
         if self.__repo_url is None:
             return
-        parsed = giturlparse.parse(self.__repo_url)
+        regex = re.compile(
```

Comment:
Please provide as a comment what is the expected URL string, much easier to read the regex.